packages/editor/src/components/editor-history/redo.js

```diff
@@ -25,8 +25,7 @@ function EditorHistoryRedo( props, ref ) {
 	const { redo } = useDispatch( editorStore );
 	return (
 		<Button
-			// TODO: Switch to `true` (40px size) if possible
-			__next40pxDefaultSize={ false }
+			__next40pxDefaultSize
```

Comment:
<table>
<tr>
<td>Before</td>
<td>After</td>
</tr>
  <tr>
    <td>
<img width="527" alt="undo-redo before" src="https://github.com/user-attachments/assets/83a89cce-c456-463f-9088-6def1c327818">

</td>
    <td>
<img width="592" alt="undo-redo after" src="https://github.com/user-attachments/assets/2414dd0c-b129-4dd4-a380-98e4d314dcb7">

</td>
  </tr>
</table>


Style didn't change because there are explicit styles for Undo and Redo buttons in code. But adding `__next40pxDefaultSize` will ensure that it will not be listed in lint next time

Reply:
Note for posterity: In our app code, sizes are added by the consumers.

https://github.com/WordPress/gutenberg/blob/9523fb4a6d568f34a217c9cab45f29413c4904f7/packages/editor/src/components/document-tools/index.js#L151-L156